edgelet/edgelet-docker/test/linux/sample_settings.tg.filepaths.yaml

```diff
@@ -1,13 +1,12 @@
-
 # Configures the provisioning mode
 provisioning:
   source: "manual"
   device_connection_string: "HostName=something.something.com;DeviceId=something;SharedAccessKey=QXp1cmUgSW9UIEVkZ2U="
 
 certificates:
-  device_ca_cert: "device_ca_cert.pem"
-  device_ca_pk: "device_ca_pk.pem"
-  trusted_ca_certs: "trusted_ca_certs.pem"
+  device_ca_cert: "/tmp/device_ca_cert.pem"
```

Comment:
I don't think we ever wanted to support relative paths. However, I don't think we had any enforcement. Our docs + samples have always provided guidance that these be absolute.

Reply:
As discussed offline with Mike, if we did support paths relative to homedir (I assume we did, since we don't manipulate the path so it would make it all the way to the OS unchanged), then there could be users relying on it. We shouldn't break those users.

Reply:
Take a look and let me know if this addresses your concern. I modified the tests as well to capture this comapt requirement.